main.js

```diff
@@ -23,6 +23,40 @@ function createWindow() {
   // Open the DevTools.
   // win.webContents.openDevTools();
 
+  // Create menu
```

Comment:
Really big `main.js` files are kinda an Electron antipattern. Remember that you can import something like this from elsewhere if you want to keep this modular!

Reply:
Awesome, I'll pull out the menu template! Thanks!